src/views/tutorial-view/components/next-previous/index.tsx

```diff
@@ -58,7 +58,7 @@ export function NextPrevious({
       return (
         <DirectionalLinkBox
           label="Next"
-          link={{ href: tutorial.next.path }}
+          href={tutorial.next.path}
```

Comment:
I've switched the props interface here, to avoid `as` as it's no longer needed.

In [previous commits I tried supporting `as` in both `CardLink` and `MaybeInternalLink`](https://github.com/hashicorp/dev-portal/commit/83f4a08799abbb785a5bd7f7a8c49d97bd2b0ffa). This worked okay, but felt overly complex given that we no longer need to use the `as` prop. Definitely open to re-adding it in future iterations if we see a need for it.

Reply:
I think we maybe used to need the `as` because of query param issues in Learn, but now we should be ok 👍